BLUESPAWN-client/BLUESPAWN-client.vcxproj

```diff
@@ -302,7 +329,55 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>Secur32.lib;DbgHelp.lib;Wintrust.lib;ws2_32.lib;Crypt32.lib;Shlwapi.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <EnableUAC Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+      </EnableUAC>
+      <UACExecutionLevel Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+      </UACExecutionLevel>
+      <UACUIAccess Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+      </UACUIAccess>
+      <EnableUAC Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
+      </EnableUAC>
+      <UACExecutionLevel Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
+      </UACExecutionLevel>
+      <UACUIAccess Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
+      </UACUIAccess>
+      <EnableUAC Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+      </EnableUAC>
+      <UACExecutionLevel Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+      </UACExecutionLevel>
+      <UACUIAccess Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+      </UACUIAccess>
+      <EnableUAC Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+      </EnableUAC>
+      <UACExecutionLevel Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+      </UACExecutionLevel>
+      <UACUIAccess Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+      </UACUIAccess>
     </Link>
+    <PostBuildEvent>

```

Comment:
Remove the condition statement and make this more concise

Reply:
fixed